scripts/create_istio_system.sh

```diff
@@ -33,3 +33,14 @@ else
 fi
 istioctl manifest generate -i $ISTIO_NAMESPACE ${ISTIO_FILES[@]/#/-f } > /app/generated-manifest.yaml
 istioctl verify-install -i $ISTIO_NAMESPACE -f /app/generated-manifest.yaml
+
+if kubectl get namespace cattle-dashboards; then
```

Comment:
Do we want to add the ability to enable / disable this from the istio chart, or do we think it should be available only based on monitoring?

Reply:
I think they can always be added if rancher-monitoring is installed. At least I can't think of a use case where you would not want to have them. In the end it's "just" some more additional dashboards in Grafana to the one that rancher-monitoring already adds there. But maybe I'm missing something :).

Reply:
@shpwrck what do you think? Should this be configurable?

Reply:
The scenario I was thinking of is if someone installs their own monitoring, not using the rancher-monitoring charts, would we expect them to install these dashboards separately?

Reply:
I would say yes. If they install their own monitoring, we don't know how it was set up. Are they using Prometheus? Are they using prometheus-operator? Are they using the Grafana? Are they using the Grafana sidecar to automatically add dashboards from ConfigMaps? How is this sidecar configured, which labels and namespaces is it looking for?

Reply:
Default Monitoring Use: Have them enabled by default
Custom Monitoring Use: Refer to docs (Add reference to istio addons in docs)